chain/network/src/routing/ibf_peer_set.rs

```diff
@@ -185,12 +186,12 @@ mod test {
         let e2 = SimpleEdge::new(p1, p2, 3);
 
         let mut sm = SlotMap::default();
-        assert_eq!(0_u64, sm.insert(&e0).unwrap());
+        assert_eq!(0_u64, sm.insert(&e0).with_context(|| format!("failed with {:?}", &e0))?);
```

Comment:
You don't need anyhow in tests, you can just supply the panic message: 

```rust
assert_eq!(Some(0_u64), sm.insert(&e0), "failed to insert {:?}", e0);
```

In general, in tests its preferable to use panics, as that's guaranteed to give you the right backtrace.

Reply:
@matklad Thanks, that works even better. I ready found a use for it: https://github.com/near/nearcore/pull/5711
I didn't realize, that by returning an error, the stack doesn't get printed, that's kind of sad.